projects/plugins/protect/src/js/routes/scan/scan-admin-section-hero.tsx

```diff
@@ -59,6 +59,23 @@ const ScanAdminSectionHero: React.FC = () => {
 		lastCheckedLocalTimestamp = new Date( status.lastChecked + ' UTC' ).getTime();
 	}
 
+	let heading = __( "Don't worry about a thing", 'jetpack-protect' );
+	if ( numThreats > 0 ) {
+		if ( hasPlan ) {
+			heading = sprintf(
+				/* translators: %s: Total number of threats */
+				__( '%1$s active threats', 'jetpack-protect' ),
```

Comment:
I think we'll want to use `_n` here cases that this wont address (eg. `1 active threats`)? Similarly in the vulnerabilities translation.

Reply:
Totally, fixed via https://github.com/Automattic/jetpack/pull/40516/commits/03dd6b3ee8339bf4bc218c2099bce3b7f386ae37 :+1: